src/styles/_variables.scss

```diff
@@ -17,6 +17,11 @@ $blue: #2b3b57 !default;
 $green: #5aae4a !default;
 $orange: #e9a95b !default;
 
+$dark-danger-main: #ec0041 !default;
```

Comment:
Maybe it is better to use a color names, what do you think (red-dark here)?
Btw, $red and $dark-danger-main look really close to each other, do we need both of them?

Reply:
I was going off the figma names when naming the colors and deciding on their hex values, I think it's best to keep them the same name as in figma so it's more obvious if they need to be reused in the future.

Reply:
The problem is that it doesn't fit into the current paradigm where we have color names and not states colors correspond to. I would ask to change the figma names then to have the same approach.